pkgs/mitsuhiko/rye/registry.yaml

```diff
@@ -0,0 +1,34 @@
+packages:
+  - type: github_release
+    repo_owner: mitsuhiko
+    repo_name: rye
+    description: An Experimental Package Management Solution for Python
+    asset: rye-{{.Arch}}-{{.OS}}.{{.Format}}
+    files:
+      - name: rye
+        src: /rye-{{.Arch}}-{{.OS}}
+    format: gz
+    replacements:
+      amd64: x86_64
+      darwin: macos
+      arm64: aarch64
+    overrides:
+      - goos: windows
+        format: raw
+        asset: rye-{{.Arch}}-{{.OS}}.exe
+        replacements:
+          amd64: x86_64
+          arm64: x86
+        checksum:
+          type: github_release
+          asset: rye-{{.Arch}}-{{.OS}}.exe.sha256
+          algorithm: sha256
+    supported_envs:
+      - darwin
+      - linux
+      - amd64
+      - arm64
```

Comment:
https://aquaproj.github.io/docs/reference/registry-config/supported-envs/

This is strange.

- `darwin` = `darwin/{amd64,arm64}`
- `amd64` = `{windows,linux,darwin}/amd64`